src/bugsnag.js

```diff
@@ -410,6 +410,63 @@
     enhance(history, "replaceState", wrapBuilder(buildReplaceState));
   }
 
+  function trackWorkers() {
+    if(!getBreadcrumbSetting("autoBreadcrumbsWorkers", true)) {
+      return;
+    }
+
+    var workers = 0;
+
+    window.Worker = hijackConstructor(window.Worker, function(url) {
```

Comment:
I'm worried hijacking this constructor might be a little too invasive. We do it for console.log and I already don't like how that affects line numbers. We're considering creating an "examples" folder in the repo or a cookbook wiki page to demonstrate advanced customizations and breadcrumb use cases.